ds_autotuning_prototype/hf_trainer_api/language-modeling/requirements.txt

```diff
@@ -0,0 +1,13 @@
+setuptools==59.5.0
+accelerate>=0.12.0
+torch>=1.5.0
+deepspeed==0.9.1
```

Comment:
Is this valid/necessary with our `0.8.3` images?

Reply:
oh no, I think that was for testing something w/ Cong's example